src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -1636,29 +1641,37 @@ private static bool IsAzureDevOpsArtifactsUrl(string location)
 
                         using (var inStream = await response.Content.ReadAsStreamAsync())
                         {
-                            downloadOutput.Append($"  {sourceUri} => {targetFile}...");
+                            downloadOutput.AppendLine($"    {sourceUri} =>");
+                            foreach (string targetFile in targetFiles)
+                            {
+                                downloadOutput.AppendLine($"      {targetFile} =>");
```

Comment:
What is the => in this one pointing to?
This looks like it'll be something like

    sourceUri =>
        target1 =>
        target2 =>
        tagetn =>

Reply:
From source uri to each output target file locally.

Reply:
Yeah, I mean the arrows at the end of each local file, they doesn't seem to be pointing anywhere.

Reply:
Oh, duh.